package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_most_searched_website_on-google",
+  "version": "1.0.0",
+  "description": "Idea title : Most googled website in 2023",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
Consider adding more details to the package.json file. For example, you could include the author's name and any relevant keywords for the project. Additionally, consider adding scripts for common tasks, such as starting the server or running tests. [medium]